pkgs/by-name/vd/vdhcoapp/package.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, fetchFromGitHub
+, buildNpmPackage
+, toml2json
+, nodejs
+, ffmpeg
+, callPackage
+, filepicker ? callPackage ./filepicker.nix {}
```

Comment:
Why is that callPackage needed?

Reply:
because filepicker is not in nixpkgs and i chose to not add it as its only used by vdhcoapp

Reply:
i am not sure if "only used by vdhcoapp" should be the requirement to not expose that as general library, but i definetly see that it might not be very useful atm for others, so i guess that could be still okay. Resolved from my side

Reply:
This will fall apart as soon as another program is added to nixpkgs named filepicker which I am not sure if that will be caught in review.

Reply:
yeah moving it to a let statement or directly adding it to nixpkgs might be a better idea.